daeun/두-수의-차.js

```diff
@@ -0,0 +1,4 @@
+function solution(num1, num2) {
+    var answer = num1 - num2;
```

Comment:
```suggestion
    const answer = num1 - num2;
```
- var 키워드 삭제하고 재할당하지 않으니 const로 변경했습니다.

Reply:
확인했습니다 감사합니다!